x-pack/test/reporting_api_integration/reporting/csv_saved_search.ts

```diff
@@ -126,12 +117,13 @@ export default function ({ getService }: FtrProviderContext) {
 
         expect(resStatus).to.eql(200);
         expect(resType).to.eql('text/csv');
-        expect(resText).to.eql(CSV_RESULT_SCRIPTED);
+        expect(resText).to.eql(fixtures.CSV_RESULT_SCRIPTED);
 
-        await esArchiver.unload('reporting/scripted_small');
+        await esArchiver.unload('reporting/scripted_small2');
       });
 
-      it('Formatted date_nanos data', async () => {
+      // BUG: date_nanos formatter does not use time zone param
+      it.skip('Formatted date_nanos data, UTC timezone', async () => {
```

Comment:
TODO: need to file an issue with @elastic/kibana-app-arch on this.

The server-side date_nanos formatter can not accept timezone as a parameter.

Reply:
https://github.com/elastic/kibana/issues/56153#issuecomment-652713466